docs/features/techdocs/addons.md

```diff
@@ -0,0 +1,214 @@
+---
+id: addons
+title: TechDocs Addons
+description: How to find, use, or create TechDocs Addons.
+---
+
+> Note: This page contains documentation for features that have not yet been
+> released on a [main-line version](https://backstage.io/docs/overview/versioning-policy#main-release-line)
+> of Backstage and TechDocs. We plan to make Addons generally available in
+> Backstage v1.2.
+
+## Concepts
+
+TechDocs is a centralized platform for publishing, viewing, and discovering
+technical documentation across an entire organization. It's a solid foundation!
+But it doesn't solve higher-order documentation needs on its own: how do you
+create and reinforce a culture of documentation? How do you build trust in the
+quality of technical documentation?
+
+TechDocs Addons are a mechanism by which you can customize the TechDocs
+experience in order to try and address some of these higher-order needs.
+
+### Addons
+
+An Addon is just a react component. Like any react component, it can retrieve
+and render data using normal Backstage or native hooks, APIs, and components.
+Props can be used to configure its behavior, where appropriate.
+
+### Locations
```

Comment:
I saw some image in a previous demo showing where the addon would be rendered, that would be a nice addition!

Reply:
💡